src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,38 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
 
     public void getStatistic(String fromFileName, String toFileName) {
-
+        int supply = 0;
+        int buy = 0;
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
+            String line = bufferedReader.readLine();
+            while (line != null) {
+                String[] split = line.split(",");
+                if (split[0].equals("supply")) {
```

Comment:
The previous remark has not been corrected, and also
![image](https://user-images.githubusercontent.com/12864128/187029036-21ca39a5-a8c2-4625-a182-b0c4bae09ba2.png)

Reply:
correct this